test/test_automl/test_estimators.py

```diff
@@ -260,7 +261,24 @@ def test_type_of_target(mock_estimator):
                     "binary targets")
 
 
-def test_cv_results(tmp_dir):
+def test_performance_over_time_no_ensemble(tmp_dir, output_dir):
+    X_train, Y_train, X_test, Y_test = putil.get_dataset('iris')
+
+    cls = AutoSklearnClassifier(time_left_for_this_task=30,
+                                per_run_time_limit=5,
+                                tmp_folder=tmp_dir,
+                                output_folder=output_dir,
+                                seed=1,
+                                initial_configurations_via_metalearning=0,
```

Comment:
We no longer use the `output_folder` option, you'll need to remove this and the fixture `output_dir` in the tests parameter.